src/proxy/index.js

```diff
@@ -77,6 +79,8 @@ export default class Proxy extends Router {
         this.POST('/messaging', (req, res, serverInfo) => this._onServiceMessage(req, res, serverInfo));
         this.GET('/task.js', (req, res, serverInfo) => this._onTaskScriptRequest(req, res, serverInfo, false));
         this.GET('/iframe-task.js', (req, res, serverInfo) => this._onTaskScriptRequest(req, res, serverInfo, true));
+        this.GET('/about:blank', (req, res) => this._onServicePageRequest(req, res, SERVICE_PAGE_TYPE.blank));
```

Comment:
This is not correct. This pages should be dispatched via proxy. So I will be able to get proxy URL fro them. But shouldn't perform any requests to server, instead it should return empty page (with no content), but we should be able to extract session id, etc.